### ISSUE TITLE
set num threads

### ISSUE THREAD

**Issue by tangchen2** [nmslib/hnswlib#335]
if i didnt't use "set_num_threads", hnsw index would use default threads values which i printed(num_threads) were 96, why the default num_threads were 96? is it set according to the system?

**Comment by yurymalkov** [comment]
Yes, it is set to the maximum number of threads. I'll add a note in the readme. Thanks!
